pkg/printers/table/collector.go

```diff
@@ -285,6 +296,36 @@ func (r *ResourceStateCollector) LatestState() *ResourceState {
 	}
 }
 
+// Stats returns a summary of the results from the actuation operation
+// as a stats.Stats object.
+func (r *ResourceStateCollector) Stats() stats.Stats {
+	var s stats.Stats
+	for _, res := range r.resourceInfos {
+		switch res.ResourceAction {
+		case event.ApplyAction:
+			if res.Error != nil {
+				s.ApplyStats.IncFailed()
+			} else {
+				s.ApplyStats.Inc(res.ApplyOpResult)
```

Comment:
are the operations not set when it's an error?

Reply:
They should be set to the default value for the operation type. I have updated the code so we handle this in the `Inc` function rather than in a separate `else` block.

Reply:
huh... looks like i actually made an operation for ReconcileFailed, but the other event operations don't have them. I wonder if we should add them in anther PR, just to be consistent. Then we could just use the operation for stats collection.

Reply:
Hmm.. I'm not sure what is the best approach here. Having a separate `Apply/Prune/DeleteFailed` operation means we potentially have two place to look. So is there a way those operations can fail without the event containing an error (I don't think so..) and is there any situation where we might want to include an error in the event, even if the operation is something else than `Failed`?
I don't feel very strongly either way.